openedx/features/partners/g2a/views.py

```diff
@@ -21,6 +21,7 @@
 from openedx.core.djangoapps.user_authn.cookies import set_logged_in_cookies
 from openedx.core.djangoapps.user_authn.views.register import REGISTER_USER, record_registration_attributions
 from openedx.features.partners.helpers import get_partner_recommended_courses
+from openedx.features.partners.models import PartnerUser
```

Comment:
@zahid-arbisoft one more thing at line number 31, I guess it would be more descriptive if you import Constants by name. like  `from .constants import * PARTNER_CONSTANTS (or any other descriptive name)`

Reply:
done